### PR TITLE
Add namespace propagated waiting check in the SynchronizedBeforeSuite

### LINKED ISSUE #1724: E2E test failed
**What happened**:
e2e test failed, this is an inevitable problem in my machine. The namespace created in karmada control plane is not synchronized to the member cluster in time.
图片

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Karmada version:
- kubectl-karmada or karmadactl version (the result of `kubectl-karmada version` or `karmadactl version`):
- Others:

### PR DESCRIPTION
**Pull request by duanmengkk**
**What type of PR is this?**

/kind bug

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #1724

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by wuyingjun-lucky** [comment]
https://github.com/karmada-io/karmada/blob/master/docs/contributors/guide/github-workflow.md 
Here is the method to squash commits

File: test/e2e/suite_test.go [hunk 1]
```diff
@@ -107,6 +107,7 @@ var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
 
 	testNamespace = fmt.Sprintf("karmadatest-%s", rand.String(RandomStrLength))
 	err = setupTestNamespace(testNamespace, kubeClient)
+	framework.WaitNamespacePresentOnClusters(framework.ClusterNames(), testNamespace)
```
Comment (XiShanYongYe-Chang) [inline test/e2e/suite_test.go:110]: We can move this line down one line. The following line is used to check whether the namespace is successfully created.

**Review by XiShanYongYe-Chang — COMMENTED** [review]
/lgtm

cc @RainbowMango

**Comment by XiShanYongYe-Chang** [comment]
Hi @duanmengkk, can you help to modify the title more specifically.

**Review by RainbowMango — APPROVED** [review]
/approve

**Comment by RainbowMango** [comment]
> Hi @duanmengkk, can you help to modify the title more specifically.

Not a big deal. Anyone can use `/retitle` command to update the title.

**Comment by XiShanYongYe-Chang** [comment]
/retitle Add namespace propagated waiting check in the SynchronizedBeforeSuite

**Pull request merged**